conf/application.conf

```diff
@@ -86,7 +86,7 @@ microservice {
       save-movement-uri = "/save-movement-submission"
       submit-consolidation = "/consolidations/submit"
       fetch-notifications = "/notifications"
-      fetch-submissions = "/submissions"
+      fetch-movements = "/movements"
```

Comment:
You need to change endpoint in movements backend

Reply:
`movements` is the correct one :)